src/libsystemd-network/sd-ipv4ll.c

```diff
@@ -263,7 +263,7 @@ static int ipv4ll_pick_address(sd_ipv4ll *ll) {
 
 #define MAC_HASH_KEY SD_ID128_MAKE(df,04,22,98,3f,ad,14,52,f9,87,2e,d1,9c,70,e2,f2)
 
-static int ipv4ll_start_internal(sd_ipv4ll *ll, bool reset_generation) {
+static int ipv4ll_start_internal(sd_ipv4ll *ll, const struct in_addr *start_address, bool reset_generation) {
```

Comment:
Why not use `sd_ipv4ll_set_address()`?? I do not think it is necessary to make this function take the starting address.